tools/scripts/mcufla.sh

```diff
@@ -33,17 +33,18 @@
 # THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 USAGE=$(cat <<-END
-Flash maxim microcontrollers using Maxim's own openocd.
-Only use .elf files, won't work with .hex files.
+Flash maxim, stm32 and pico microcontrollers, it downloads specific openocd for Maxim and STM32
+platform, for Pico standard openocd is required.
+Can be used only with .elf files.
 
 Dependencies (Debian):
-	autoconf automake make git libudev-dev libtool libusb-1.0-0-dev pkg-config texinfo
+	autoconf automake make git libudev-dev libtool libusb-1.0-0-dev pkg-config texinfo openocd
 
 Usage:
 	maxfla.sh path/to/binary.elf [adapter_serial_number]
 
 Author:
-	darius.berghe@analog.com
+	darius.berghe@analog.com and andreea.andrisan@analog.com
```

Comment:
Add your name on a new line. There might be other people contributing in the future and it's better formatted that way.